apps/condo/domains/marketplace/components/MarketplacePaymentsContent.tsx

```diff
@@ -141,15 +153,21 @@ const MarketplacePaymentsTableContent = () => {
         fetchPolicy: 'network-only',
     })
 
-    const { data: allPaymentsSum, loading: allPaymentsSumLoading } = usePaymentsSum({ ...searchPaymentsQuery })
-    const { data: donePaymentsSum, loading: donePaymentsLoading } = usePaymentsSum({ ...searchPaymentsQuery, status: PAYMENT_DONE_STATUS })
-    const { data: inProcessPaymentsSum, loading: inProcessPaymentsLoading } = usePaymentsSum({ ...searchPaymentsQuery, status_in: [PAYMENT_PROCESSING_STATUS, PAYMENT_WITHDRAWN_STATUS, PAYMENT_ERROR_STATUS] })
+    const { data: allPaymentsSum, loading: allPaymentsSumLoading } = usePaymentsSum({ ...searchPaymentsQuery, status_in: [ PAYMENT_WITHDRAWN_STATUS, PAYMENT_DONE_STATUS] })
+    const { data: donePaymentsSum, loading: donePaymentsLoading } = usePaymentsSum({ ...searchPaymentsQuery, status_in: [PAYMENT_DONE_STATUS] })
+    const { data: inProcessPaymentsSum, loading: inProcessPaymentsLoading } = usePaymentsSum({ ...searchPaymentsQuery, status_in: [PAYMENT_WITHDRAWN_STATUS] })
```

Comment:
you try get <code>in<b>Process</b>PaymentsSum</code>, but use in filter 

<code>PAYMENT_<i>WITHDRAWN</i>_STATUS</code> instead of 

<code>PAYMENT_<b>PROCESSING</b>_STATUS</code>